### PR TITLE
Chunter: Remove attachments only for creator

### LINKED ISSUE #1526: Chunter: Remove attachments only for creator

### PR DESCRIPTION
**Pull request by denis-bunakalya**
closes https://github.com/hcengineering/anticrm/issues/1526

### CODE REVIEW

File: plugins/attachment-resources/src/components/FileBrowser.svelte [hunk 1]
```diff
@@ -282,11 +284,17 @@
           <div class="item flex">
             <AttachmentPresenter value={attachment} />
           </div>
-          <div class="eAttachmentRowActions" class:fixed={i === selectedFileNumber}>
-            <div id="context-menu" class="eAttachmentRowMenu" on:click={(event) => showFileMenu(event, attachment, i)}>
-              <IconMoreV size={'small'} />
+          {#if attachment.modifiedBy !== myAccId}
```
Comment (BykhovDenis) [inline plugins/attachment-resources/src/components/FileBrowser.svelte]: Menu can contains other actions, you should hide action, not menu

**Review by BykhovDenis — CHANGES_REQUESTED** [review]

File: plugins/attachment-resources/src/components/FileBrowser.svelte (hunk 1 above)
Comment (denis-bunakalya) [reply plugins/attachment-resources/src/components/FileBrowser.svelte]: Done

File: plugins/attachment-resources/src/components/FileBrowser.svelte [hunk 2]
```diff
@@ -56,7 +56,7 @@
 
   $: fetch(searchQuery, selectedSort, selectedFileTypeId, selectedDateId, selectedParticipants, selectedSpaces)
 
-  async function fetch (
+  async function fetch(
```
Comment (BykhovDenis) [inline plugins/attachment-resources/src/components/FileBrowser.svelte]: Why you remove space here?

**Review by BykhovDenis — APPROVED** [review]

File: plugins/attachment-resources/src/components/FileBrowser.svelte (hunk 2 above)
Comment (denis-bunakalya) [reply plugins/attachment-resources/src/components/FileBrowser.svelte]: Reverted

**Pull request merged**